apps/preview/next/pages/showcases/Pagination/Pagination.tsx

```diff
@@ -134,7 +134,7 @@ export function Showcase() {
             >
               <button
                 type="button"
-                className="px-4 py-3 rounded-md text-neutral-500 hover:bg-primary-100 hover:text-primary-800 active:bg-primary-200 active:text-primary-900 "
+                className="px-4 py-3 rounded-md text-neutral-500 w-auto hover:bg-primary-100 hover:text-primary-800 active:bg-primary-200 active:text-primary-900 "
```

Comment:
question: this button always has auto width? if so do we need this class I guess it will be default behavior